packages/next-s3-upload/package.json

```diff
@@ -60,6 +60,7 @@
     "@aws-sdk/client-s3": ">=3.427.0",
     "@aws-sdk/client-sts": "^3.427.0",
     "@aws-sdk/lib-storage": ">=3.427.0",
+    "@aws-sdk/s3-presigned-post": "3.427.0",
```

Comment:
I had to require the exact version, otherwise two versions of `S3Client` live in the project.